sample/todo-sync/src/com/cloudant/todo/TasksModel.java

```diff
@@ -72,7 +72,7 @@ public TasksModel(Context context) {
         );
         DatastoreManager manager = DatastoreManager.getInstance(path.getAbsolutePath());
         try {
-            this.mDatastore = manager.openDatastore(TASKS_DATASTORE_NAME);
+            this.mCloudantSync = manager.openDatastore(TASKS_DATASTORE_NAME);
```

Comment:
I don't think `openDatastore` is a great naming thing now that its called `CloudantSync`. Nothing to worry about for this PR though.